### PR TITLE
fixing issue when the data is empty

### PR DESCRIPTION
**Pull request by madhuv-msft**

### CODE REVIEW

File: src/AzureDevOpsKubeService.ts [hunk 1]
```diff
@@ -17,17 +17,35 @@ export class AzureDevOpsKubeService extends KubeServiceBase {
     fetch(resourceType: KubeResourceType): Promise<any> {
         switch (resourceType) {
             case KubeResourceType.Pods:
-                return Promise.resolve(JSON.parse(this._kubeSummaryData.pods));
+                return resolvePromise(this._kubeSummaryData.pods);
```
Comment (rakol-ms) [inline src/AzureDevOpsKubeService.ts]: Can this be `Promise.resolve(safeParseJson(this._kubeSummaryData.pods));` ?

Comment (madhuv-msft) [reply src/AzureDevOpsKubeService.ts]: i want to avoid writing Promise.resolve(safeParseJson()) too many times, so did this way.

Comment (savenkat) [reply src/AzureDevOpsKubeService.ts]: I tend to agree with rajendra here as it isn't evident from `resolvePromise(this._kubeSummaryData.pods)` that we are actually parsing the json and that too in a fail-safe manner.

Comment (madhuv-msft) [reply src/AzureDevOpsKubeService.ts]: fixed as per the commend of Rajendra.

**Pull request merged**